modules/acquia_cms_headless/acquia_cms_headless.info.yml

```diff
@@ -2,20 +2,20 @@ name: "Acquia CMS Headless"
 package: 'Acquia CMS'
 type: module
 description: 'Provides functionality for Progressively Decoupled and Purely Headless sites using Node and Next JS.'
-core_version_requirement: ^9
+core_version_requirement: ^9 || ^10
 dependencies:
-  - drupal:acquia_cms_tour
-  - drupal:consumers
-  - drupal:entity_clone
-  - drupal:jsonapi_extras
-  - drupal:jsonapi_menu_items
-  - drupal:jsonapi_resources
-  - drupal:next
-  - drupal:next_jsonapi
-  - drupal:openapi_jsonapi
-  - drupal:openapi_ui
-  - drupal:openapi_ui_redoc
-  - drupal:openapi_ui_swagger
+  - acquia_cms_tour:acquia_cms_tour
+  - modules:consumers
```

Comment:
@chandan-singh7929 Is it correct i.e `modules:consumers` ??
cc: @rajeshreeputra

Reply:
this needs to be updated to `consumers:consumers`, thanks @vishalkhode1 for pointing this out.

Reply:
Done